akka-docs/src/test/scala/docs/io/EchoServer.scala

```diff
@@ -10,32 +10,10 @@ import scala.concurrent.duration.DurationInt
 
 import com.typesafe.config.ConfigFactory
 
-import akka.actor.{ Actor, ActorDSL, ActorLogging, ActorRef, ActorSystem, Props, SupervisorStrategy }
-import akka.actor.ActorDSL.inbox
+import akka.actor.{ Actor, ActorLogging, ActorRef, ActorSystem, Props, SupervisorStrategy }
 import akka.io.{ IO, Tcp }
 import akka.util.ByteString
 
-object EchoServer extends App {
```

Comment:
Perhaps we should rewrite this one instead of removing?

Reply:
:+1: wasn't included in the docs but useful to have a main users can run if they go to source